components/autocalc/autocalcspell.cpp

```diff
@@ -0,0 +1,250 @@
+#include "autocalcspell.hpp"
+
+#include <climits>
+#include <cfloat>
+#include <set>
+
+#include <components/esm/attr.hpp>
+#include <components/esm/loadmgef.hpp>
+#include <components/esm/loadrace.hpp>
+#include <components/esm/loadclas.hpp>
+#include <components/esm/loadspel.hpp>
+#include <components/esm/loadnpc.hpp>
+
+#include "autocalc.hpp"
+
+// Most of the code in this file was moved from apps/openmw/mwmechanics/autocalcspell.cpp
```

Comment:
Can we remove this comment (and the similar one in autocalc.cpp)? Anyone is free to browse the repository for history lessons.